client/src/app/site/pages/organization/pages/committees/pages/committee-detail/components/committee-detail/committee-detail.component.ts

```diff
@@ -21,5 +24,6 @@ export class CommitteeDetailComponent extends BaseModelRequestHandlerComponent {
         if (+params[`committeeId`]) {
             this.subscribeTo(getCommitteeDetailSubscriptionConfig(+params[`committeeId`]), { hideWhenDestroyed: true });
         }
+        this.subscribeTo(getCommitteeMeetingDetailExternalIdsSubscriptionConfig());
```

Comment:
Maybe we could wait with opening the subscription until the dialog is opened and just do a single request there.

Reply:
The isUniqueAmoung validator needs an array at the creation of the dialog.

Reply:
Added a version with the external ids fetched before the dialog is called.